ms-patches/reduce-allocation-merges.yml

```diff
@@ -0,0 +1,14 @@
+title: Reduce Allocation Merges
+- work_item: 1612900 
+- jbs_bug: JDK-8287061
+- author: JohnTortugo
```

Comment:
maybe JohnTortugo (<msft-alias>) so we can find you

Reply:
I think that's not a requirement, right? I'll refrain from doing that since it may break some current (or future) automation that isn't expecting that kind of input.